Source/WebKit/WebProcess/WebPage/ViewGestureGeometryCollector.h

```diff
@@ -41,17 +42,23 @@ namespace WebKit {
 
 class WebPage;
 
-class ViewGestureGeometryCollector : private IPC::MessageReceiver {
+class ViewGestureGeometryCollector : private IPC::MessageReceiver, public RefCounted<ViewGestureGeometryCollector> {
```

Comment:
The `WeakRef<WebPage>` data member is no longer safe now that this is ref-counted.